### PR TITLE
OSDOCS-11201: updates attributes file in MicroShift

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.17+

Issue:
[OSDOCS-11201](https://issues.redhat.com/browse/OSDOCS-11201)

Link to docs preview:
1. All the changes made under installing for the rhde compatibility matrix can be seen below:
[RPM package](https://78509--ocpdocs-pr.netlify.app/microshift/latest/microshift_install/microshift-install-rpm#microshift-install-rpm-rhde-compatibility-table)
[RHEL for EDGE image](https://78509--ocpdocs-pr.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree#embed-ostree-rhde-compatibility-table)
[offline use](https://78509--ocpdocs-pr.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree-offline-use#embed-offline-rhde-compatibility-table)

2. All the changes made under updating  can be seen in [About updates](https://78509--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-about-updates)

QE review:
- [x] QE has approved this change.
- [x] SME has approved this change.

### CODE REVIEW

**Comment by aravipra** [comment]
WIP donot close

**Comment by aravipra** [comment]
can users update for 4.15 to 4.17?

File: snippets/microshift-rhde-compatibility-table-snip.adoc [hunk 1]
```diff
@@ -16,6 +16,11 @@
 ^|*{microshift-short} Release Status*
 ^|*Supported {microshift-short} Version&#8594;{microshift-short} Version Updates*
 
+^|9.4
+^|4.17
+^|Generally Available
+^|4.17.0&#8594;4.17.z, 4.16&#8594;4.17, 4.14&#8594;4.16 and 4.15&#8594;4.16
```
Comment (DanielFroehlich) [inline snippets/microshift-rhde-compatibility-table-snip.adoc:22]:
```suggestion
^|4.17.0&#8594;4.17.z, 4.16&#8594;4.17
```

Comment (aravipra) [reply snippets/microshift-rhde-compatibility-table-snip.adoc]: done

**Comment by aravipra** [comment]
@DanielFroehlich  Is there any chnages required under the "Important " section in the link https://78509--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-update-options

**Comment by DanielFroehlich** [comment]
No, that "important" section is still correct.
But seeing the rendered version, I notice that  "Checking version update path"-> "MicroShift version 4.16" is missing a bullet for a V4.16 to V4.17 update statement.

**Comment by aravipra** [comment]
> No, that "important" section is still correct. But seeing the rendered version, I notice that "Checking version update path"-> "MicroShift version 4.16" is missing a bullet for a V4.16 to V4.17 update statement.

done.

File: microshift_updating/microshift-update-rpms-manually.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-Updating {product-title} for non-OSTree systems such as {op-system-base-full} requires downloading then updating the RPMs. For patch releases, such as 4.16.1 to 4.16.2, download and update the RPMs. For minor-version release updates, add the step of enabling the update repository using your subscription manager.
+Updating {product-title} for non-OSTree systems such as {op-system-base-full} requires downloading then updating the RPMs. For patch releases, such as 4.17.1 to 4.17.2, download and update the RPMs. For minor-version release updates, add the step of enabling the update repository using your subscription manager.
 
 [IMPORTANT]
 ====
```
Comment (jogeo) [inline microshift_updating/microshift-update-rpms-manually.adoc:12]: There are conflicting statements in the "Manual updates with RPMs" section. Both are in IMPORTANT boxes.
The first and correct text is:
"MicroShift updates are supported with a maximum of two minor versions. For example, you can update to 4.16 from 4.14 in a single step. Updating 4.14 to 4.15 first is not required."

The second farther down on the same page says the contrary:
"You can only update MicroShift from one version to the next in sequence. Jumping minor versions is not supported. For example, must update 4.15 to 4.16."

Comment (aravipra) [reply microshift_updating/microshift-update-rpms-manually.adoc:12]: @DanielFroehlich should the important section in the link https://78509--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-update-rpms-manually#microshift-updating-rpms_microshift-update-rpms-manually be modified or removed?

Comment (DanielFroehlich) [reply microshift_updating/microshift-update-rpms-manually.adoc:12]: good catch @jogeo !
@aravipra , Yes, please remove the second father down Important box. That is a left over from the early days.

Comment (aravipra) [reply microshift_updating/microshift-update-rpms-manually.adoc:12]: done

**Review by DanielFroehlich — APPROVED** [review]
/LGTM

File: microshift_updating/microshift-about-updates.adoc [hunk 3]
```diff
@@ -6,11 +6,11 @@ include::_attributes/attributes-microshift.adoc[]
...
 toc::[]
 
-Updates are supported on {product-title} beginning with the General Availability version 4.14. With the 4.16 release, the following updates are supported:
+Updates are supported on {product-title} beginning with the General Availability version 4.14. With the 4.17 release, the following updates are supported:
 
 * A maximum of two minor versions to the next in sequence, for example, from 4.14 to 4.16.
-* One minor version to the next in sequence, for example, from 4.15 to 4.16.
-* Patch updates are also supported from z-stream to z-stream, for example 4.16.1 to 4.16.2.
+* One minor version to the next in sequence, for example, from 4.16 to 4.17.
+* Patch updates are also supported from z-stream to z-stream, for example 4.17.1 to 4.17.2.
```
Comment (mburke5678) [inline microshift_updating/microshift-about-updates.adoc:13]: This is likely the way it has always been done, but this makes me think the 4.17.1 and 4.17.2 versions are available already.

**Comment by mburke5678** [comment]
@aravipra One comment, feel free to disregard. Otherwise LGTM

File: microshift_updating/microshift-about-updates.adoc (hunk 3 above)
Comment (aravipra) [reply microshift_updating/microshift-about-updates.adoc:13]: The versions mentioned are an example. So can be retained as is.

File: microshift_updating/microshift-about-updates.adoc [hunk 4]
```diff
@@ -6,11 +6,11 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-Updates are supported on {product-title} beginning with the General Availability version 4.14. With the 4.16 release, the following updates are supported:
+Updates are supported on {product-title} beginning with the General Availability version 4.14. With the 4.17 release, the following updates are supported:
 
 * A maximum of two minor versions to the next in sequence, for example, from 4.14 to 4.16.
-* One minor version to the next in sequence, for example, from 4.15 to 4.16.
-* Patch updates are also supported from z-stream to z-stream, for example 4.16.1 to 4.16.2.
+* One minor version to the next in sequence, for example, from 4.16 to 4.17.
```
Comment (michaelryanpeter) [inline microshift_updating/microshift-about-updates.adoc:12]: You might want to consider making a new variable, such as `ocp-version-n1` (similar to [this in the `common-attributes` file](https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc#L207-L209)). If you create an attribute for the previous version of OCP/MicroShift/etc, you could update these version numbers once in the attributes file and be done with it.

Definitely not a blocker, but something to consider if you want to simplify the maintenance of these example versions.

**Review by michaelryanpeter — COMMENTED** [review]
I left a comment to consider if you want to make maintaining this doc a little easier going forward. Feel free to ignore it if you don't find it useful.

File: microshift_updating/microshift-about-updates.adoc (hunk 4 above)
Comment (aravipra) [reply microshift_updating/microshift-about-updates.adoc:12]: will check if this can be done. Thank you for the suggestion

**Pull request merged**